### PR TITLE
📦 - Add build system.

### LINKED ISSUE #49: Rollup integration
Use webpack to manage bundling, minification, and library definitions.

### LINKED ISSUE #59: Clean up .npmignore

### PR DESCRIPTION
**Pull request by arb**
Add rollbar build script. Closes #49.
Cleaned up some stuff in .npmignore. Closes #59.

### CODE REVIEW

File: scripts/build.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            plugins,
+          })
+          .then(bundle => {
+            console.log(chalk.green(`${config.name} successfully rolled.`));
+            console.log(chalk.yellow(`Starting write for ${config.name}...`));
+            return bundle.write({
+              format: 'umd',
+              dest: config.dest,
+              // Inject the version here because the JSON plugin here pulls everything in and freezes it, thus adding a lot of bloat.
+              // intro: `global.VERSION = '${pkg.version}';`,
```
Comment (tysoeh) [inline scripts/build.js]: Do you want this commented?

File: scripts/build.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    );
+  })
+  .then(() => {
+    console.log(
+      chalk.bgBlack.greenBright(
+        'All Stardog.js builds completeled successfully.'
+      )
+    );
+  })
+  .catch(e => {
```
Comment (tysoeh) [inline scripts/build.js]: I mean, if you don't wanna be slick... ;)

File: scripts/build.js (hunk 1 above)
Comment (arb) [inline scripts/build.js]: Yeah I want to leave that in there so I don't have to figure out an alternative way to do it in the future.

File: scripts/build.js (hunk 2 above)
Comment (arb) [inline scripts/build.js]: So you *DO* learn from me!

**Pull request merged**